content/en/watchdog/insights.md

```diff
@@ -0,0 +1,113 @@
+
+Each insight highlights one outlier or anomaly affecting a subset of users. Depending on the product area, Watchdog Insights displays different types of anomalies. Examples include, but are not limited to, the following:
+- Error and latency outliers in logs, traces, and RUM views
+- Spike in error logs
+- New error logs
+- Deadlocked threads
+- High percentage of unready Kubernetes pods
+
+### Prioritization
+
+Watchdog sorts insights based on a combination of factors to place the most important insight at the beginning of the list. The factors that Watchdog takes into account can include the following:
+- State (ongoing versus resolved)
+- Status (warning, error, or critical)
+- Start time
+- Anomaly type
+
+## Navigation
+
+{{< img src="watchdog/log_explorer_watchdog_insights.png" alt="The Watchdog Insights banner on the Logs Explorer, showing three anomalies: new error logs in the web-store service, a spike in error logs in the product-recommendation service, and another spike in error logs in the product-recommendation service" >}}
+
+The Watchdog Insights banner sits near the top of each page. Expand the banner for an overview. The highest priority insights appear on the left. If Watchdog cannot find any issues, the banner is gray.
+
+### Filter on Insight
+
+To refine your current view to match a Watchdog Insight, hover over the top right corner of an insight summary card. Two icons appear. Click on the inverted triangle icon with the tooltip **Filter on Insight**. The page refreshes to show a list of entries corresponding to the insight.
+
+### Side panel
+
+Click **View all** to expand the panel. A side panel opens from the right, containing a vertical list of Watchdog Insights. Each entry shows a detailed view, with more information than the summary card.
+
+{{< img src="watchdog/log_explorer_watchdog_insights_panel.png" alt="The Watchdog Insights side panel view within Logs Explorer. The top area shows a bar graph of error statuses over time. One Log Anomaly card is visible with the title 'New error logs were detected on service:web-store'" >}}
+
+### Detailed view
+
+For a detailed view of an insight, click on the individual card. The full side panel opens from the right.
+
+{{< img src="watchdog/profiler_watchdog_insight.png" alt="A Watchdog Insights full side panel view, with the title 'Lock Pressure is high in service:product-recommendation'" >}}
+
+To share an insight in one click, click the **Copy Link** button on the full side panel. Your clipboard populates with the query that produced the insight.
+
+## Locations within Datadog
+
+Four areas within Datadog feature Watchdog Insights: Infrastructure, APM, Log Management, and RUM.
```

Comment:
Can we add the in-app links for these areas? 
https://app.datadoghq.com/apm/home
https://app.datadoghq.com/orchestration/overview/pod
https://app.datadoghq.com/logs, https://app.datadoghq.com/rum/explorer
```suggestion
You can find Watchdog Insights in four product areas: Infrastructure, APM, Log Management, and RUM.
```